src/profile-logic/convert-markers.js

```diff
@@ -73,12 +69,18 @@ export function upgradeGCMajorMarker(marker: Object) {
         timings.slices = timings.slices.length;
       }
 
-      timings.phase_times = _upgradePhaseTimes(timings.totals);
+      if (upgrade_processed) {
+        timings.phase_times = convertPhaseTimes(timings.totals);
+      } else {
+        timings.phase_times = timings.totals;
+      }
       delete timings.totals;
 
-      timings.mmu_20ms /= 100;
-      timings.mmu_50ms /= 100;
-      timings.allocated *= 1024 * 1024;
+      if (upgrade_processed) {
+        timings.mmu_20ms /= 100;
+        timings.mmu_50ms /= 100;
+        timings.allocated *= 1024 * 1024;
```

Comment:
given we now have `allocated_bytes` in the latest iteration of your gecko patch, do we still want to change `allocated` ?

Reply:
I'll probably have to check this myself.  thanks.

Reply:
Yeah, this was weird. I'll update it to make the upgraded processed profile use allocated_bytes and remove allocated from the processed profiles.